### ISSUE TITLE
A single missing Twitter post may cause many posts not to display

### ISSUE THREAD

**Issue by samsm-turner** [ampproject/amphtml#2530]
We have some articles that use many Twitter embeds in the process of telling a story. If one of these Twitter posts has been deleted, the failure seems to affect several other posts as well.

So for example, on a page with the following Twitter embeds:
```
valid1
valid2
missing1
valid3 
valid4
```
missing1 doesn't appear (which makes sense) but valid3 and valid4 also might not load.

[Here's screenshot](http://samsm.com/whatever/skitch/Screenshot_3_10_16__8_58_AM_1C91BAC5.png)

The error is: 
```
Uncaught (in promise) TypeError {message: "Cannot read property 'contentWindow' of null"}message: "Cannot read property 'contentWindow' of null" twitter.js:55
```

Here's [an example](http://samsm.com/whatever/amp-twitter-example.html) of the sort of markup we are using. 
623637080355270656 is the missing post id (after "Imma let you finish but"). The rest in that document should be good. Quickly scrolling to that part of the page seems to usually reproduce the issue, although occasionally only the missing Tweet will be missing.

We are going to attack this from the other side and try to not publish deleted Tweets, but it seems inevitable that some will briefly sneak through.

Thanks for reading. Please let me know if you have any questions.

Sam

**Comment by cramforce** [comment]
I took a look and I really cannot see how this could effect multiple tweets (unless they were replies of the same tweet in a single embed). Nonetheless, we need to handle this case much better. PR incoming.

**Comment by samsm-turner** [comment]
You are correct, I misinterpreted the problem. Only the one Tweet is effected, though the effect I was seeing took up much more space which I mistook for multiple Tweets.

Looks like a great addition, suspect it fixes my problem.

**Comment by jridgewell** [comment]
@cramforce: Is this fixed by #2537?

**Comment by cramforce** [comment]
@jridgewell Not 100% sure, but lets me optimistic.
